app/src/pages/project/TracesTable.tsx

```diff
@@ -91,6 +94,59 @@ type NestedSpanTableRow<TSpan extends IAdditionalSpansRow> = TSpan & {
   children: NestedSpanTableRow<TSpan>[];
 };
 
+const TableBody = <
```

Comment:
Direct copy of the previous tbody content, minus new styles in the `td` element to prevent wrapping.